lib/Controller/AppointmentConfigController.php

```diff
@@ -22,72 +23,67 @@
  * License along with this library.  If not, see <http://www.gnu.org/licenses/>.
  *
  */
+
 namespace OCA\Calendar\Controller;
 
 use OCA\Calendar\Db\AppointmentConfig;
+use OCA\Calendar\Exception\ClientException;
 use OCA\Calendar\Exception\ServiceException;
 use OCA\Calendar\Http\JsonResponse;
 use OCA\Calendar\Service\Appointments\AppointmentConfigService;
 use OCP\AppFramework\Controller;
-use OCP\AppFramework\Http\TemplateResponse;
-use OCP\IInitialStateService;
+use OCP\AppFramework\Http;
 use OCP\IRequest;
-use OCP\IUser;
 
-/**
- * Class PublicViewController
- *
- * @package OCA\Calendar\Controller
- */
 class AppointmentConfigController extends Controller {
 
-	/** @var IInitialStateService */
-	private $initialStateService;
-
-	/** @var IUser */
-	private $user;
```

Comment:
The user should stay. Currently, the parameter `$userId` is allowed to be null in the constructor but later in the `create()` method it is required to be non-null -> might go :boom:

Reply:
I will add a null check.